test/jest/integration/integration.spec.ts

```diff
@@ -1,16 +1,13 @@
 import "../../../jest"
-import {filesOfProject} from "../../../src/files/fluentapi/files";
+import { filesOfProject } from "../../../src/files/fluentapi/files"
 
 describe("Integration test", () => {
-
 	it("checks the created messages", async () => {
-
 		const rule = (await filesOfProject(__dirname + "/samples/filenamingsample/tsconfig.json"))
 			.inFolder("services")
 			.should()
-			.matchPattern(".*Service\.ts")
+			.matchPattern(".*Service.ts")
```

Comment:
Is this correct? Shouldn't the dot be double-escaped (once for regex, and once to escape the backslash in the string)? Otherwise "aServicebts" would match.

```suggestion
			.matchPattern(".*Service\\.ts")
```

Reply:
I agree, but should we change this in this PR?